modules/serverless-functions-adding-annotations.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-annotations.adoc
+
+[id="serverless-functions-adding-annotations_{context}"]
+= Adding annotations to a function
+
+.Procedure
+
+. Open the `func.yaml` file for your function.
+
+. For every annotation that you want to add, add the following YAML to the `annotations` section:
```

Comment:
```suggestion
. You can add Kubernetes annotations to a deployed function, by adding them to the `annotations` section in func.yaml. For example, to indicate that a function was authored by Alice, you might include the following annotation.
```

I suggest a little more contextual information with this introduction.

Reply:
Added this info, albeit slightly differently than you suggested.